proposals/0022-multi-stake.md

```diff
@@ -0,0 +1,183 @@
+---
+simd: '0022'
+title: Multi Delegation Stake Account
+authors:
+  - Jon Cinque (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-01-20
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+A new "multi stake" account that allows for multiple movements of stake, useful
+for simpler stake movement through more activations, deactivations, or redelegation.
+
+## New Terminology
+
+* "multi stake": a stake account with multiple `Delegation` instances
+* "small stake": a stake whose delegation is a small lamport amount
```

Comment:
Is this really some arbitrary small stake or is it directly related to the current minimum stake delegation amount?

Reply:
Technically it's supposed to be anything under the minimum delegation amount, but until that's approved, we can just say it's anything under 1 SOL?

Reply:
Imo we should define as "less than current minimum stake delegation amount". Today it is 0, but in the future that may be updated. This would make sense in the context of the Motivation section

Reply:
Done, and to be pedantically clear, it's actually 1 lamport currently!